documentation/modules/ova-prerequisites.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: REFERENCE
+[id="ova-prerequisites_{context}"]
+= Open Virtual Appliance (OVA) prerequisites
+
+The following prerequisites apply to Open Virtual Appliance (OVA) file migrations:
+
+* All OVA files are created by VMware vSphere.
+
+[NOTE]
+====
+Migration of OVA files that were not created by VMware vSphere but are compatible with  vSphere might succeed. However, migration of such files is not supported by {project-short}. {project-short} supports only OVA files created by VMware vSphere.
+====
+
+* The OVA files are in one or more folders under an NFS shared directory.
```

Comment:
@RichardHoch 
Isn't this bullet redundant? The next bullet seems to cover it.

Reply:
@sbeskin-redhat There are two options for the folders under the NFS directory  1) OVAs in compressed OVF packages and 2) in extracted folders. But option 1 has two sub-options.

Reply:
@sbeskin-redhat Will the words "as follows:" help? 
 
 Requirement 1. All files created by vSphere
Requirement 2: OVA files in NFS share as follows:
Option 1. Compressed OVF packages
Option 1a: This structure
Option 1b: That structure
Option 2: Extracted folders

Reply:
@RichardHoch 
But that's more or less what line 19 already says. Just remove line 18. 
You may replace "in one of the following structures" with "as follows" if you like.

Reply:
@sbeskin-redhat I need to specify they are under an NFS share. Only line 18 mentions that.

Reply:
@RichardHoch 
But line 19 says "The OVA files are stored in the NFS shared directory". How is it different?
Just keep one of those lines.